packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -462,6 +470,25 @@ contract Deploy is Deployer {
         console.log("New safe: %s deployed at %s\n    Note that this safe is owned by the deployer key", _name, addr_);
     }
 
+    /// @notice If the keepDeployer option was used with deploySafe(), this function can be used to remove the deployer.
+    ///         address (msg.sender).
+    ///         Note this function does not have the broadcast modifier.
+    function removeDeployerFromSafe(string memory _name, uint256 _newThreshold) public {
+        Safe safe = Safe(mustGetAddress(_name));
+
+        // The sentinel address is used to mark the start and end of the linked list of owners in the Safe.
+        address sentinelOwners = address(0x1);
+
+        // Because deploySafe() always adds msg.sender first (if keepDeployer is true), we know that the previousOwner
+        // will be sentinelOwners.
```

Comment:
We should have a test for this in case `deploySafe()` is ever refactored

Reply:
The existing tests will break if that happens.

Reply:
The high-level invariant this comment expresses is: "if `keepDeployer` is used to deploy a new Safe, then `removeDeployerFromSafe()` will work to remove the deployer. 

The refactoring that would most likely break this is if `msg.sender` was added to the owners array in some location other than index-0, though it would break not due to a lack of private keys, rather because it would become much more difficult to identify the correct `prevOwner` to provide (the first arg to `removeOwner` [here](https://github.com/ethereum-optimism/optimism/blob/maur/ownership-refac-3/packages/contracts-bedrock/scripts/Deploy.s.sol#L487)). 

If at some point in the future we embark on a project to test all of the various utility functions in `Deploy.s.sol`, then this test would make sense to add, otherwise it feels like an odd one-off. 

Another reason that I think we are OK, is that we make assertions on the final set of owners in [`_checkSafeConfig()`](https://github.com/ethereum-optimism/optimism/blob/maur/ownership-refac-3/packages/contracts-bedrock/test/Safe/DeployOwnership.t.sol#L34). 

If you like though, I can add `assertFalse(_safe.isOwner(msg.sender))` to that function, to ensure that we're never leaving the deployer address on the Safe.